repo/packages/D/dcos-aws-cli/0/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "packagingVersion": "3.0",
+  "name": "dcos-aws-cli",
+  "version": "1.0",
+  "maintainer": "joerg@mesosphere.io",
+  "minDcosReleaseVersion": "1.9",
+  "description": "DC/OS AWS CLI",
```

Comment:
Add `preInstallNotes` and add this line to it: `This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!`